packages/chain-adapters/src/evm/avalanche/AvalancheChainAdapter.ts

```diff
@@ -33,16 +33,21 @@ export class ChainAdapter extends EVMBaseAdapter<KnownChainIds.AvalancheMainnet>
     super({ chainId: DEFAULT_CHAIN_ID, supportedChainIds: SUPPORTED_CHAIN_IDS, ...args })
 
     this.assetId = avalancheAssetId
+    this.rpcUrl = args.rpcUrl
     this.parser = new unchained.ethereum.TransactionParser({
       chainId: this.chainId,
-      rpcUrl: args.rpcUrl
+      rpcUrl: this.rpcUrl
     })
   }
 
   getType(): KnownChainIds.AvalancheMainnet {
     return KnownChainIds.AvalancheMainnet
   }
 
+  getRpcUrl(): string {
```

Comment:
Can just add to the `EVMBaseAdapter`

Reply:
Addressed in https://github.com/shapeshift/lib/pull/851/commits/d85a1d5864b02754a403fc8601803466105deac7